framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -271,8 +275,8 @@ def run_local():
             return json.dumps(e, cls=c_common.WazuhJSONEncoder)
         except exception.WazuhInternalError as e:
             e.dapi_errors = self.get_error_info(e)
-            # Avoid exception info if it is an asyncio timeout
-            self.logger.error(f"{e.message}", exc_info=True if e.code != 3021 else False)
+            # Avoid exception info if it is an asyncio timeout or JSONDecodeError
+            self.logger.error(f"{e.message}", exc_info=True if e.code not in [3021, 3036] else False)
```

Comment:
```suggestion
            self.logger.error(f"{e.message}", exc_info=e.code not in {3021, 3036})
```

Reply:
Adding **suggestion + change of try/except to a higher level** in a single commit